test/tests.zig

```diff
@@ -695,15 +695,15 @@ pub fn addCliTests(b: *std.Build) *Step {
         // Test `zig init-lib`.
         const tmp_path = b.makeTempPath();
         const init_lib = b.addSystemCommand(&.{ b.zig_exe, "init-lib" });
-        init_lib.cwd = tmp_path;
+        init_lib.setCwd(.{ .cwd_relative = tmp_path });
```

Comment:
This is incorrect, it should be relative to the build root, not the current working directory.

```suggestion
        init_lib.setCwd(.{ .path = tmp_path });
```

Reply:
I used `cwd_relative` as it's supposed to be used when dealing with absolute paths and `makeTempPath` has this doc comment stating that it returns an absolute path
```
/// This function is intended to be called in the `configure` phase only.
/// It returns an absolute directory path, which is potentially going to be a
/// source of API breakage in the future, so keep that in mind when using this
/// function.
```